backend/src/api/files.py

```
@@ -42,17 +50,49 @@ class FileStatsResponse(BaseModel):
+    enable_quality_check: bool = Form(True),
+    custom_metadata: Optional[str] = Form(None),
+    current_user: User = Depends(get_current_user),  # Temporarily reduced permission check
     db: Session = Depends(get_db),
     file_service: FileService = Depends(get_file_service)
 ):
     """Upload a file to the system"""
+
+    # Debug logging
+    import logging
+    logger = logging.getLogger(__name__)
+    logger.info(f"📤 Upload Request Debug:")
```

Ruff (0.14.1)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)